integration-tests/go.mod

```diff
@@ -256,7 +256,7 @@ require (
 	github.com/gogo/protobuf v1.3.3 // indirect
 	github.com/gogo/status v1.1.1 // indirect
 	github.com/golang-jwt/jwt/v4 v4.5.1 // indirect
```

Comment:
Need to bump this  as well. https://github.com/advisories/GHSA-mh63-6h87-95cp

Affected versions:  < 4.5.2, >= 5.0.0-rc.1, < 5.2.2

Patched versions: 4.5.2, 5.2.2